target/linux/ramips/dts/PBR-D1.dts

```diff
@@ -43,7 +43,7 @@
 		#size-cells = <0>;
 		poll-interval = <20>;
 
-		wps {
+		reset {
```

Comment:
Albeit the change of the node name is something pure cosmetical, the whole node still looks wrong.

If it's a reset button, the ```linux,code``` should be ```KEY_RESTART```.

If it's a WPS Button, the ```label``` should be ```wps``` or similar (depending on how it is named in the manual or what is printed on the case).